htdocs/search.php

```diff
@@ -2133,7 +2139,7 @@ function outputSearchForm($options)
         }
     }
     sql_free_result($rsAttrGroup);
-    if ($bBeginLine2 == false) {
+    if (!$bBeginLine2) {
```

Comment:
My preference would be `false === $bBeginLine2` - to make it clear what type is expected.

Reply:
done